src/libtilt/interpolation/interpolate_dft_3d.py

```diff
@@ -49,7 +49,8 @@ def sample_dft_3d(
     samples = torch.view_as_complex(samples.contiguous())  # (b, )
 
     # pack data back up and return
-    [samples] = einops.unpack(samples, pattern='*', packed_shapes=ps)
+    # [samples] = einops.unpack(samples, pattern='*', packed_shapes=ps)
+    samples = samples.reshape(*ps) #Ask Alister if this will work in any situation
```

Comment:
interesting, is this a significant performance gain? I don't think this works in the general case but it should always work here as far as I can tell

Reply:
ohhh I see - torch compile couldn't go through unpack?

Reply:
Exactly. This is a requirement for the compiler